CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+project(ProTouchApiSimulator)

```

Comment:
Readme has to be changed after dynamic_gui_tester.py is removed

Reply:
adjusted